### ISSUE TITLE
Result page(s)

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit/dashboard#71]
- Create a nice view of a single result. (We already have a table/list of them at https://dashboard.stg.packit.dev/jobs).
- Basically, it should be a nice-looking variant of the old result page `packit-service` is providing.

Here is a list of information I would like to see for all job types:
- Project name and link to the upstream
- Information about upstream trigger (PR, branch, release) and link to it.
- Human readable datetime.

**In the case of SRPM, add:**
- Logs
- *(old version: https://stg.packit.dev/srpm-build/3070/logs)*

**In the case of Copr build, add:**
- SRPM logs / link to the SRPM view
- Link to download SRPM file
- Link to Copr project web UI (e.g. https://copr.fedorainfracloud.org/coprs/packit/packit-hello-world-231-stg/build/1954610/)
- Link to Copr logs (e.g. https://copr-be.cloud.fedoraproject.org/results/packit/packit-hello-world-231-stg/fedora-33-x86_64/01954610-hello/builder-live.log.gz)
- Install instructions.
- *(old version: https://stg.packit.dev/copr-build/8830)*

**In the case of Koji build, add:**
- SRPM logs / link to the SRPM view
- Link to Koji web UI
- Link to the logs page
- *(old version: https://prod.packit.dev/koji-build/59)*

**In the case of Testing Farm result, add:**
- Link to the Copr Build view
- Some status, basic result description
- Link to the TF dashboard/result page
- *No alternative in the service.*

I've described all the pages so we have a bigger picture and see if something can be shared/connected.

**Comment by IceWreck** [comment]
So we need an API endpoint which provides all these details in JSON. Something like 
* `/srpm-builds/{db_id}` which provides the same info in JSON as https://stg.packit.dev/srpm-build/3070/logs 
* `/koji-builds/{db_id}` for https://prod.packit.dev/koji-build/59
* `/copr-builds/{db_id}`  for https://stg.packit.dev/copr-build/8830

In these example links, the integers are db_ids.

Note we already have an **unused** API endpoint at `/copr-builds/{id}` which provides some of this information. Here `{id}` is the build_id from copr instead of the id of a row in CoprBuildModel. Similarly the endpoint for `/koji-builds/{id}` is also **unused.** Maybe we should replace these with the ones mentioned above.

Basically use our own DB's ids for the API endpoints instead of relying on the ones provided by copr or koji.

I don't have packit service setup somewhere right now so can you please make these endpoints ? I'll then make the dashboard pages for these and we can add a details button in each row of the jobs view which links to the details page.

The details page will also be independently accessible at dashboard.packit.dev/jobs/copr-builds/{db_id}, dashboard.packit.dev/jobs/srpm-builds/{db_id} and so on.

**Comment by lachmanfrantisek** [comment]
@IceWreck Yes, you're right. Here is an issue for that in the packit-service repo. I hope someone will take a look soon.

packit/packit-service#966

**Comment by lachmanfrantisek** [comment]
@IceWreck API is now ready on stage (https://stg.packit.dev/api/) if you want to try it.

Let us know if there is some problem with the format/values. Thanks!

**Comment by IceWreck** [comment]
Will work on the results page tomorrow (Monday)

**Comment by lachmanfrantisek** [comment]
The PR is merged, I think we can close it. If there are some problems, we can create new well-defined issue.
